src/ripple/protocol/impl/STTx.cpp

```diff
@@ -178,8 +186,7 @@ Blob STTx::getSignature () const
 
 void STTx::sign (RippleAddress const& private_key)
 {
-    Blob signature;
-    private_key.accountPrivateSign (getSigningHash (), signature);
```

Comment:
Hahahahahahha! This caller didn't even check the return value I was so worried about in the definition of `accountPrivateSign`. :laughing: